packages/babel-generator/test/fixtures/comments/create-parenthesized-expressions/options.json

```diff
@@ -0,0 +1,3 @@
+{
+  "parserOpts": {"createParenthesizedExpressions": true}
```

Comment:
This works because I updated the runner in packages/babel-generator/test/index.js to spread the parserOpts into the parser options.